easybuild/easyconfigs/g/Gtk-Doc/Gtk-Doc-1.26-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Gtk-Doc'
+version = '1.26'
+
+homepage = 'https://www.gtk.org/gtk-doc'
+description = "GTK-Doc is a project which was started to generate API documentation from comments added to C code."
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+
+source_urls = ['https://download.gnome.org/sources/gtk-doc/%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['bff3f44467b1d39775e94fad545f050faa7e8d68dc6a31aef5024ba3c2d7f2b7']
+
+builddependencies = [
+    ('binutils', '2.27'),
+    ('ITSTool', '2.0.2'),
+]
+
```

Comment:
Hmm, looks like it needs http://docbook.org/xml/4.3/ + libxml (for `xmlcatalog`).

Do you have any `docbook` RPMs installed @verdurin?

```
$ rpm -qa | grep docbook
docbook-style-xsl-1.78.1-3.el7.noarch
docbook-dtds-1.0-60.el7.noarch
```